keyboards/keychron/q1/rev_0102/keymaps/victorsavu3/keymap.c

```diff
@@ -0,0 +1,153 @@
+     KC_LSFT,  KC_NUBS,  KC_Z,     KC_X,     KC_C,     KC_V,     KC_B,     KC_N,     KC_M,     KC_COMM,  KC_DOT,   KC_SLSH,            KC_RSFT,  KC_UP,
+     KC_LCTL,  KC_LGUI,  KC_LALT,                                KC_SPC,                                 KC_RALT, MO(WIN_FN),KC_RCTL,  KC_LEFT,  KC_DOWN,  KC_RGHT),
+
+[WIN_FN] = LAYOUT_iso_83(
+     KC_TRNS,            KC_BRID,  KC_BRIU,  KC_CALC,  KC_FLXP,  RGB_VAD,  RGB_VAI,  KC_TRNS,  KC_PSCR,  KC_MUTE,  KC_VOLD,  KC_VOLU,  KC_MPLY,  KC_TRNS,  KC_TRNS,
+     KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,            KC_TRNS,
+     RGB_TOG,  RGB_MOD,  RGB_VAI,  RGB_HUI,  RGB_SAI,  RGB_SPI,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,                      KC_TRNS,
+     KC_CAPS,  RGB_RMOD, RGB_VAD,  RGB_HUD,  RGB_SAD,  RGB_SPD,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,            KC_TRNS,
+     KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,            KC_TRNS,  KC_PGUP,
+     KC_TRNS,  KC_TRNS,  KC_TRNS,                                KC_TRNS,                                KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_HOME,  KC_PGDN,  KC_END)
+
+};
+
+static bool wiggle_mouse;
+static uint16_t wiggle_timer;
+static uint16_t next_wiggle;
+
+
+LEADER_EXTERNS();
+
+void matrix_scan_user(void) {
+    LEADER_DICTIONARY() {
+        leading = false;
+        leader_end();
+
+        SEQ_ONE_KEY(KC_LEAD) {
+            register_code(KC_CAPS);
+            unregister_code(KC_CAPS);
+        }
+
+        SEQ_FOUR_KEYS(KC_I, KC_D, KC_L, KC_E) {
+            wiggle_mouse = !wiggle_mouse;
+            wiggle_timer = timer_read();
+        }
+
+        SEQ_TWO_KEYS(KC_O, KC_K) {
+            send_unicode_string("👍");
+        }
+
+        SEQ_THREE_KEYS(KC_S, KC_A, KC_D) {
+            send_unicode_string("😞");
+        }
+
+        SEQ_FIVE_KEYS(KC_C, KC_H, KC_E, KC_C, KC_K) {
+            send_unicode_string("✅");
+        }
+
+        SEQ_FIVE_KEYS(KC_C, KC_R, KC_O, KC_S, KC_S) {
+            send_unicode_string("❎");
+        }
+
+        SEQ_FIVE_KEYS(KC_T, KC_H, KC_A, KC_N, KC_K) {
+            send_unicode_string("🙏");
+        }
+
+        SEQ_FIVE_KEYS(KC_S, KC_M, KC_I, KC_L, KC_E) {
+            send_unicode_string("😊");
+        }
+
+        SEQ_FIVE_KEYS(KC_P, KC_A, KC_R, KC_T, KC_Y) {
+            send_unicode_string("🎉");
+        }
+
+        SEQ_FOUR_KEYS(KC_E, KC_Y, KC_E, KC_S) {
+            send_unicode_string("(ಠ_ಠ)");
+        }
+
+        SEQ_FIVE_KEYS(KC_M, KC_A, KC_G, KC_I, KC_C) {
+            send_unicode_string("(ಠ_ಠ) 🪄 ⠁⭒*.✫.*⭒⠁");
+        }
+
+        SEQ_FIVE_KEYS(KC_T, KC_A, KC_B, KC_L, KC_E) {
+            send_unicode_string("(ノಠ痊ಠ)ノ彡┻━┻");
+        }
+
+        SEQ_FIVE_KEYS(KC_S, KC_H, KC_R, KC_U, KC_G) {
+            send_unicode_string(R"(¯\_(ツ)_/¯)");
```

Comment:
```suggestion
            send_unicode_string(R"(¯\\_(ツ)_/¯)");
```
Backslash needs to be escaped.